src/conversion/convertJsDateToSasDate.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Converts a JavaScript date object to a SAS Date or Datetime, given the logic below:
+ *
+ * A JS Date contains the number of _milliseconds_ since 01/01/1970
+ * A SAS Date contains the number of _days_ since 01/01/1960
+ * A SAS Datetime contains the number of _seconds_ since 01/01/1960
+ *
+ * @param jsDate JS Date to be converted. The type is instance of `Date`
+ * @param unit Unit in which to return the SAS Date / datetime, eg `sasdate | sasdatetime`
+ * @returns SAS Date value based on `unit` param
+ */
+export const convertJsDateToSasDate = (
+  jsDate: Date,
+  unit: 'sasdate' | 'sasdatetime'
+): number => {
+  let valueInMilliseconds: number = 0
+
+  valueInMilliseconds = new Date(
```

Comment:
no need for  above declaration of variable at line # 16 just  add const here before variable